### DISCUSSION TITLE
How to Deploy A Static HTML Website?  Is it possible?

### DISCUSSION THREAD

**Discussion by Honkenbonkers** [caprover/caprover#1734] [General]
Hello, 

I've been messing around with CapRover for a bit. I'm fairly new to all this (Docker, CapRover, Linux, code in general ... lol) but I'm learning more every day.

**I was wondering if there was a way to deploy and host simple html websites using CapRover?**

- Is there a way to use docker to host static html sites in a container?
- How would I deploy, edit, and configure something like that? (I'm not afraid of ssh-ing into my server and messing around, haha) 
- Could I shell into that container to edit?

Let me know if something like that is even possible (yet?) or if I'm missing something. 

Thanks!, R

**Comment by githubsaturn** [comment]
Absolutely possible! There are a million ways of doing so. Basically you want to bundle a webserver with your plain html files in a container.

Think of containers as virtual machines. You run a webserver (nginx/apache, etc) in a container and all your html files can be inside a container.

A very simple example is to use our PHP template. Ignore the fact that it's called php. It's just an apache server (with the ability to run php).
- Create a git repo
- Create a file in the root of the repo as `index.html`
- Create a file called `captain-definition` and put this content inside the file:
```
{
 "schemaVersion" :2 ,
 "templateId" :"php/7.1.10"
}
```
- Now run git commit 
- Then run `caprover deploy` 

Another method is [described here](https://caprover.com/docs/recipe-deploy-create-react-app.html).
